walk/config.go

```diff
@@ -104,6 +112,46 @@ func (cr *Configurer) Configure(c *config.Config, rel string, f *rule.File) {
 	c.Exts[walkName] = wcCopy
 }
 
+func (c *Configurer) loadBazelIgnore(repoRoot, rel string, wc *walkConfig) error {
+	// Only load .bazelignore at the root, and only once
+	if rel != "" {
+		return nil
+	}
```

Comment:
Ah so rel is effectively empty for the rest of the func.

Worth to note this detail as context for the rest